### PR TITLE
docs: Add notes about email confirmation for Spotify login

### PR DESCRIPTION
**Pull request by dshukertjr**
Screenshot 2024-02-01 at 16 58 28

### CODE REVIEW

**Review by J0 — APPROVED** [review]

File: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx [hunk 1]
```diff
@@ -52,6 +52,14 @@ Under `Redirect URIs`:
 
 ## Add login code to your client app
 
+The following outlines the steps to sign-in using Spotify with Supabase Auth.
```
Comment (charislam) [inline apps/docs/pages/guides/auth/social-login/auth-spotify.mdx:55]:
```suggestion
The following outlines the steps to sign in using Spotify with Supabase Auth.
```

File: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx [hunk 2]
```diff
@@ -52,6 +52,14 @@ Under `Redirect URIs`:
...
+
+1. Call the sign-in method from the client library.
```
Comment (charislam) [inline apps/docs/pages/guides/auth/social-login/auth-spotify.mdx:57]:
```suggestion
1. Call the signin method from the client library.
```

`sign in` the verb, `signin` the noun (I guess adjective here 🤷🏼)

File: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx [hunk 3]
```diff
@@ -52,6 +52,14 @@ Under `Redirect URIs`:
...
+1. The user is redirected to the Spotify login page.
+1. After completing the sign-in process, the user will be redirected to the your app with an error that says the email address needs to be confirmed. Simultaneously the user receives a confirmation email from Supabase Auth.
```
Comment (charislam) [inline apps/docs/pages/guides/auth/social-login/auth-spotify.mdx:59]:
```suggestion
1. After completing the sign-in process, the user will be redirected to your app with an error that says the email address needs to be confirmed. Simultaneously the user receives a confirmation email from Supabase Auth.
```

File: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx [hunk 4]
```diff
@@ -52,6 +52,14 @@ Under `Redirect URIs`:
...
+1. The user clicks the confirmation link in the email.
+1. The user is brought back to the app and is not signed in.
```
Comment (charislam) [inline apps/docs/pages/guides/auth/social-login/auth-spotify.mdx:61]: `not` or `now`?

**Review by charislam — APPROVED** [review]

File: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx (hunk 4 above)
Comment (dshukertjr) [reply apps/docs/pages/guides/auth/social-login/auth-spotify.mdx:61]: Thanks, it should be `now`
```suggestion
1. The user is brought back to the app and is now signed in.
```

**Pull request merged**